### PR TITLE
Update templates to latest functions SDK

### PR DESCRIPTION
**Pull request by thechenky**
We are releasing functions SDK v3.1.0, so updating the templates in the init portion of the functions to use latest version.

### CODE REVIEW

**Comment by thechenky** [comment]
I'm really not understanding why the tests keep failing, this is cosmetic change

**Comment by samtstern** [comment]
@thechenky the hosting functional test has been really flaky today due to some 500s coming from production.  I got the same error you got on a totally unrelated PR:
```
Error: HTTP Error: 500, Request '/ApiKeys.ListApiKeys' from role 'mobilesdk-devrel-frontend' throttled: Service is overloaded (too-many-retries) go/tr-o.
```

**Comment by thechenky** [comment]
@samtstern thanks for the context!

File: changelog.txt [hunk 1]
```diff
@@ -3,4 +3,5 @@
 * Improve error message when `firebase serve` can't acquire the right port.
 * Allow running the Firestore and RTDB emulators without a configuration.
 * Allow the Firestore emulator to give more information about invalid rulesets.
-* Hot reload firestore rules on change.
\ No newline at end of file
+* Hot reload firestore rules on change.
+* Update functions init templates to v3.1.0
```
Comment (bkendall) [inline changelog.txt]:
```suggestion
* Update functions init templates to `v3.1.0`.
```

**Comment by bkendall** [comment]
I updated your branch, so you may have to pull...

**Comment by thechenky** [comment]
Got it, thanks! This should be good to go now, we released 3.1.0 earlier today.

**Pull request merged**